game-app/game-core/src/main/java/games/strategy/triplea/attachments/PlayerAttachment.java

```diff
@@ -165,19 +114,47 @@ private void resetMovementLimit() {
   }
 
   private void setAttackingLimit(final String value) throws GameParseException {
+    attackingLimit.add(parseUnitLimit("attackingLimit", value));
+  }
+
+  private void setAttackingLimit(final Set<StackingLimit> value) {
+    attackingLimit = value;
+  }
+
+  private Set<StackingLimit> getAttackingLimit() {
+    return attackingLimit;
+  }
+
+  private void resetAttackingLimit() {
+    attackingLimit = new HashSet<>();
+  }
+
+  private static class StackingLimit {
```

Comment:
(observation) A quick comment on this data type might be helpful to clarify what exactly 'StackingLimit' means. 
I agree it can be induced relatively easily by checking the data types.

Second observation, the 'all args constructor' can be omitted and replaced with lombok `@AllArgsConstructor`. Sometimes 'getters' can be nice to have (for example in function references & lambda expressions), in this case `@Value` would be a good choice. An additional benefit is if we ever chose to extract this class to its own file, there would be no updates needed. All that is to say, this is a good candidate class to have `@Value` on it.